akvo/rsr/tests/test_templatetags.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+"""Akvo RSR is covered by the GNU Affero General Public License.
+
+See more details in the license.txt file located at the root folder of the Akvo RSR module.
+For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+"""
+
+from django.test import TestCase
+
+from akvo.rsr.models import ProjectEditorValidation, ProjectEditorValidationSet
+from akvo.rsr.templatetags.project_editor import mandatory_or_hidden
+
+
+class TemplateTagssTestCase(TestCase):
+    """Tests for template tags."""
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
It woule be nice to use the actions defined in ProjectEditorValidation here, something like:
```python
def setUpClass(cls):
    MANDATORY = ProjectEditorValidation.MANDATORY_ACTION
    HIDDEN = ProjectEditorValidation.HIDDEN_ACTION
    validation1 = [
        ('rsr_project.title', MANDATORY)
        ('rsr_project.subtitle', MANDATORY),
        ...
```